docs/source/roles.md

```diff
@@ -92,7 +92,8 @@ The IT professionals and software engineers who will be responsible for developi
 ## Governance Roles
 
 Roles that uphold the governance of TREs.
-Such governance responsibilities typically involve establishing policies and procedures to ensure the responsible use of data, protecting the privacy and confidentiality of research participants, and promoting transparency and accountability in research activities.
+Such governance responsibilities typically involve establishing policies and procedures to ensure the responsible use of data.
```

Comment:
```suggestion
Governance responsibilities typically involve establishing and implementing policies and procedures to ensure the responsible use of data.
```

Some of the roles are about ensuring procedures are or have been followed rather than setting them out in the first place.